reference/features/search_parameters.md

```diff
@@ -156,55 +185,51 @@ And you would get the following response:
       "overview": "Chris and his girlfriend Rose go upstate to visit her parents for the weekend. At first, Chris reads the family's overly accommodating behavior as nervous attempts to deal with their daughter's interracial relationship, but as the weekend progresses, a series of increasingly disturbing discoveries lead him to a truth that he never could have imagined.",
     }
   ],
-  ...
+  …
   "query": "thriller"
 }
 ```
 
-## The facets distribution
-
-If you have [set up faceted attributes](/reference/features/settings.md#attributes-for-faceting), you can retrieve the count of matching terms for each [facets](/reference/features/faceted_search.md).
+## Facets distribution
 
-`facetsDistribution=[<facetName>, <facetName>, ...]`
+**Parameter**: `facetsDistribution`
+**Expected value**: an array of `facetName`s or `["*"]`
+**Default value**: `null`
 
-This attribute can take two values:
+Adds the count of matching facet terms to the query's returned fields.
 
-- `[<facetName>, <facetName>, ...]` (Array of strings)
+This argument can take two values:
 
-  An array of strings that contains the facets for which to retrieve the matching count. The number of remaining candidates for each specified facet is returned.
-  If a facet name doesn't exist, it will be ignored.
+- An array of `facetName`s—this will only count the listed facets:  `facetsDistribution=[<facetNameA>, <facetNameB>, ...]`. If a `facetName` does not exist, it will be ignored
 
-- `["*"]`
+- An asterisk—this will return a count for all facets: `["*"]`
```

Comment:
My reasoning here was that this bit would explain what `*` actually does. Most users will know, but I thought stating it explicitly could be useful for inexperienced programmers.

Reply:
You are totally right. Thanks for thinking of that 🙏🏻 
On a semi-related note, I just remembered that in order to be used in this parameter, `facetName`s must be added to `attributesForFaceting` (I believe—feel free to test).
I think we should add that information above, and also change this text to:

```
- An asterisk—this will return a count for all facets defined in `attributesForFaceting`
```